README.md

```diff
@@ -12,28 +12,28 @@ $ yarn add @chainsafe/dappeteer
 ## Usage
 
 ```js
-import puppeteer from 'puppeteer';
-import dappeteer from '@chainsafe/dappeteer';
+import D from '@chainsafe/dappeteer';
 
 async function main() {
-  const [metamask, page] = await dappeteer.bootstrap(puppeteer, { metaMaskVersion: 'v10.15.0' });
+  const { page, dappeteer } = await D.bootstrap({ metaMaskVersion: 'v10.15.0' });
```

Comment:
@mpetrunic @BeroBurny I'm not sure we're following the best path here in terms of naming. I import above `D from '@chainsafe/dappeteer'`, so as not to have a name conflict, but the fact that we return what used to be `metamask` called now `dappeteer` with the type `Dappeteer` seems to be counterintuitive. 

I think we should rename this `dappateer` to `metamask` so that we keep the api close to what it was before

Reply:
agree to rename `dappeteer` to `metaMask`, while naming in the past time we referred to commanding MetaMask to do things

Reply:
agree to rename `dappeteer` to `metaMask`, while naming in the past time we referred to commanding MetaMask to do things

Reply:
agree to rename `dappeteer` to `metaMask`, while naming in the past time we referred to commanding MetaMask to do things

Reply:
Wouldn't it make more sense to rename everything to dappeteer as we are returning dappeteer API. On the other hand, we could rename `page` to `metaMaskPage` so that it's more clear, especially now that we are dealing with notifications page and snapInstallPage

Reply:
I don't really have a strong opinion, but I'd prefer to have things like `metaMask.sign` than `dappeteer.sign` because it's closer to what it was before. Agreed to rename the `metaMaskPage` as well, I'll do that.

Reply:
makes sense^^

Reply:
could you update initSnapEnv return params as well?
Discussion related to that: #198

Reply:
Sure, I'll add all the missing doc I can find in this PR.